fabbwled-backend/src/main/resources/sections/397-432.yaml

```diff
@@ -0,0 +1,60 @@
+- number: 397
+  content:
+    - text: Your ship is thrown about like flotsam and jetsam. When the storm subsides, you take stock. Much has been swept overboard – you lose 1 Cargo Unit, if you had any, of your choice. Also, the ship has been swept way off course and the mate has no idea where you are. ‘We’re lost at sea, Cap’n!’ he moans. Turn to 90.
```

Comment:
Can you remove the `Turn to 90` everywhere?

Reply:
yessir, done